python/GangaDirac/Lib/Utilities/DiracUtilities.py

```diff
@@ -34,8 +35,14 @@ def getDiracEnv(force=False):
     lock = Dirac_Env_Lock
     lock.acquire()
     if DIRAC_ENV == {} or force:
-        if getConfig('DIRAC')['DiracEnvFile'] != "" and os.path.exists(getConfig('DIRAC')['DiracEnvFile']):
-            with open(getConfig('DIRAC')['DiracEnvFile'], 'r') as env_file:
+        config_file = getConfig('DIRAC')['DiracEnvFile']
+        if not os.path.exists(config_file):
+            absolute_path = os.path.join(os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe()))),
+                                        '../../..', config_file)
```

Comment:
At least this should probably use `os.path.dirname(os.path.realpath(__file__))` but is there not a better way to get the file rather than a relative path?